pylib/cqlshlib/sslhandling.py

```diff
@@ -48,32 +48,11 @@ def get_option(section, option):
         except configparser.Error:
             return None
 
-    def get_best_tls_protocol(ssl_ver_str):
-        # newer python versions suggest to use PROTOCOL_TLS to negotiate the highest TLS version.
-        # older protocol versions have been deprecated:
-        # https://docs.python.org/2/library/ssl.html#ssl.PROTOCOL_TLS
-        # https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLS
-        if ssl_ver_str:
-            return getattr(ssl, "PROTOCOL_%s" % ssl_ver_str, None)
```

Comment:
add here warning instead this return, remove next for loop completely and after if, just return `ssl.PROTOCOL_TLS`.